pkg/skaffold/kubernetes/manifest/gke_toleration.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2022 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manifest
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/containerd/containerd/platforms"
+	spec "github.com/opencontainers/image-spec/specs-go/v1"
+	v1 "k8s.io/api/core/v1"
+	apimachinery "k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/output/log"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/yaml"
+)
+
+const (
+	tolerationField = "tolerations"
+)
+
+var gkeARMToleration = v1.Toleration{
+	Key:      "kubernetes.io/arch",
+	Operator: v1.TolerationOpEqual,
+	Value:    "arm64",
+	Effect:   v1.TaintEffectNoSchedule,
+}
+
+// SetGKEARMToleration adds a toleration for GKE ARM node taint `kubernetes.io/arch="arm64":NoSchedule`.
+func (l *ManifestList) SetGKEARMToleration(ctx context.Context, rs ResourceSelector, platforms PodPlatforms) (ManifestList, error) {
```

Comment:
Do we have a standard naming style for writing abbreviation with Camel Case. We use upper case abbreviation in variables may cause ambiguity, especially when we have abbreviation sitting along with each other like this, GKEARM,  it's not very intuitive to see where to separate words.  Should we just treat abbreviations as lower case words, then this method name become SetGkeArmToleration, an exception might be we only use upper case abbreviation as the first word of a variable name, something like IOException, JSONParse, the words separation is also clear in this case, but in general take abbreviation as lower case words seem better.

Reply:
that's the go style guide for acronyms. See [here](go/go-style/decisions#initialisms)

Reply:
Ohh ! thank you!！